test/packages/parallel/custom_entrypoint/data_stream/agent_logs/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -0,0 +1,14 @@
+---
+description: Pipeline for processing sample logs
+processors:
+- json:
+    field: message
+    target_field: target
+- set:
+    field: error.message
+    value: not present target
+    if: ctx?.target == null
+on_failure:
+- set:
+    field: error.message
+    value: '{{ _ingest.on_failure_message }}'
```

Comment:
If message is not a JSON (written by the entrypoint script) , system tests are going to fail.